skiko/build.gradle.kts

```diff
@@ -325,7 +325,11 @@ kotlin {
                 val skiaBinDir = "$skiaDir/out/${buildType.id}-${osArch.first.id}-${osArch.second.id}"
                 kotlinOptions {
                     val linkerFlags = when (osArch.first) {
-                        OS.MacOS -> listOf("-linker-option", "-framework", "-linker-option", "Metal")
+                        OS.MacOS -> listOf("-linker-option", "-framework", "-linker-option", "Metal",
```

Comment:
may be need an automation to intersperse the "-linker-option"

Reply:
yep